Library/Formula/tomcat-native.rb

```diff
@@ -14,20 +13,20 @@ class TomcatNative < Formula
 
   depends_on "libtool" => :build
   depends_on "tomcat" => :recommended
-  depends_on :java => "1.7"
+  depends_on :java => "1.7+"
   depends_on "openssl"
   depends_on "apr" => :optional
 
   def install
     cd "jni/native" do
-      if build.with? 'apr'
-        apr_path = Formula['apr'].opt_prefix
+      if build.with? "apr"
+        apr_path = Formula["apr"].opt_prefix
       else
         apr_path = "#{MacOS.sdk_path}/usr"
       end
       system "./configure", "--prefix=#{prefix}",
                             "--with-apr=#{apr_path}",
-                            "--with-java-home=#{`/usr/libexec/java_home`.chomp}",
+                            "--with-java-home=#{`/usr/libexec/java_home -v 1.7+`.chomp}",
```

Comment:
If it's only used in build time, let's make it `"--with-java-home=#{ENV["JAVA_HOME"]}"`